manim/mobject/svg/svg_mobject.py

```diff
@@ -128,116 +140,133 @@ def generate_points(self):
         the SVGMobject's points from XML tags, populating self.mobjects, and
         any submobjects within self.mobjects.
         """
-        doc = minidom.parse(self.file_path)
-        self.ref_to_element = {}
+        doc = minidom_parse(self.file_path)
         for svg in doc.getElementsByTagName("svg"):
-            mobjects = self.get_mobjects_from(svg)
+            mobjects = self.get_mobjects_from(svg, self.DEFAULT_SVG_STYLE)
             if self.unpack_groups:
                 self.add(*mobjects)
             else:
                 self.add(*mobjects[0].submobjects)
         doc.unlink()
 
-    def get_mobjects_from(self, element):
+    def get_mobjects_from(
+        self,
+        element: MinidomElement,
+        inherited_style: Dict[str, str],
+        within_defs=False,
+    ) -> List[VMobject]:
         """Parses a given SVG element into a Mobject.
 
         Parameters
         ----------
-        element : :class:`str`
+        element : :class:`Element`
             The SVG data in the XML to be parsed.
 
+        inherited_style : `dict`
+            Dictionary of the SVG attributes for children to inherit.
+
+        within_defs : `bool`
+            Whether `element` is within a `defs` element, which indicates
```

Comment:
use double `, its rst. 
```suggestion
            Whether ``element`` is within a ``defs`` element, which indicates
```

Reply:
TIL, thanks.